models/intermediate/int_ga4_export__sessions_report.sql

```diff
@@ -1,14 +1,10 @@
-{{
+{{ 
```

Comment:
It doesn't seem like this model is used downstream at all?
![image](https://github.com/user-attachments/assets/ae873b4f-53a5-4845-ad50-a594490d0cbd)

Is this model entirely necessary?